tee-worker/litentry/core/data-providers/src/graphql.rs

```diff
@@ -97,26 +96,11 @@ impl VerifiedCredentialsIsHodlerIn {
 		VerifiedCredentialsIsHodlerIn { addresses, from_date, network, token_address, min_balance }
 	}
 
-	pub fn conv_to_string(&self) -> String {
-		let mut flat = "addresses:[".to_string();
-		for addr in self.addresses.iter() {
-			flat += &format!("\"{}\",", addr);
-		}
-		flat += "],";
-		flat += &format!("fromDate:\"{}\",", self.from_date.clone());
-		match &self.network {
-			VerifiedCredentialsNetwork::Litentry => flat += "network:litentry",
-			VerifiedCredentialsNetwork::Litmus => flat += "network:litmus",
-			VerifiedCredentialsNetwork::Polkadot => flat += "network:polkadot",
-			VerifiedCredentialsNetwork::Kusama => flat += "network:kusama",
-			VerifiedCredentialsNetwork::Khala => flat += "network:khala",
-			VerifiedCredentialsNetwork::Ethereum => flat += "network:ethereum",
-		}
-		if self.token_address.is_empty().not() {
-			flat += &format!(",tokenAddress:\"{}\"", &self.token_address.clone());
-		}
-		flat += &format!(",minimumBalance:{:?}", self.min_balance.clone());
-		flat
+	pub fn to_graphql(&self) -> String {
+		let addresses_str = format!("{:?}", self.addresses);
+		let network = format!("{:?}", self.network).to_lowercase();
+		let q = format!("VerifiedCredentialsIsHodler(addresses:{}, fromDate:\"{}\", network:{}, tokenAddress:\"{}\",minimumBalance:{:?}){{isHodler,address}}", addresses_str, self.from_date, network, self.token_address, self.min_balance);
+		format!("{{{}}}", q)
```

Comment:
format twice, making them into one is better.
```
format!(r#"{{VerifiedCredentialsIsHodler.....{{isHodler,address}}}}"#)
```

Reply:
Have corrected it